assembl/static2/js/app/graphql/RootIdeasQuery.graphql

```diff
@@ -3,11 +3,11 @@ query RootIdeasQuery($lang: String!) {
     ... on Idea {
       children {
         ... on Idea {
-          id
-          title(lang: $lang)
-          description(lang: $lang)
-          numPosts
-          numContributors
+          id,
+          title(lang: $lang),
+          description(lang: $lang),
+          numPosts,
```

Comment:
Yes, again the changes are inverted here. The queries are auto formatted with prettier without comma.